Dockerfile

```diff
@@ -0,0 +1,102 @@
+FROM python:3.10-buster
+
+ARG USERNAME=flask
+ARG USER_UID=1000
+ARG USER_GID=$USER_UID
+
+RUN mkdir code \
+  && groupadd --gid $USER_GID $USERNAME \
+  && useradd --uid $USER_UID --gid $USER_GID -m $USERNAME \
+  && chown -R ${USER_UID}:${USER_GID} /code
+
+# can upgrade R to something sensible later, once we have the package lists and versions set
+# https://github.com/rocker-org/rocker/blob/2f92c6c8b8da7b3e61aabc44cacc0439bf267d31/r-base/3.6.3/Dockerfile
+
+COPY --from=r-base:3.6.3 /usr/lib/R /usr/lib/R
+COPY --from=r-base:3.6.3 /usr/local/lib/R /usr/local/lib/R
+COPY --from=r-base:3.6.3 /usr/lib/libR.so /usr/lib/libR.so
+COPY --from=r-base:3.6.3 /usr/bin/R /usr/bin/R
+COPY --from=r-base:3.6.3 /usr/bin/r /usr/bin/r
+COPY --from=r-base:3.6.3 /usr/bin/Rscript /usr/bin/Rscript
+COPY --from=r-base:3.6.3 /lib/x86_64-linux-gnu/libreadline.so.8 /lib/x86_64-linux-gnu/libreadline.so.8
+COPY --from=r-base:3.6.3 /usr/lib/x86_64-linux-gnu/libm.so /usr/lib/x86_64-linux-gnu/libm.so
+COPY --from=r-base:3.6.3 /lib/x86_64-linux-gnu/libm.so.6 /lib/x86_64-linux-gnu/libm.so.6
+COPY --from=r-base:3.6.3 /etc/R /etc/R
+COPY --from=r-base:3.6.3 /usr/share/R /usr/share/R
```

Comment:
Looks good to me. Can confirm that our prod environment currently uses R-base 3.6.1, so this should work.